demo-pipeline/buildspec/wait-for-backup.yaml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+env:
+  exported-variables:
+    - INSTANCE_RP_ID
+phases:
+  install:
+    commands:
+      - apt update
+      - apt -yq install jq
+      - curl -OfsSL https://raw.githubusercontent.com/elastio/elastio-stack/master/scripts/install-elastio.sh
+      - chmod a+x install-elastio.sh
+      - ./install-elastio.sh -c
+  build:
+    commands:
+      - echo "**********\n*\n* Waiting for job $JOB_ID\n*\n**********\n"
+      - demo-pipeline/scripts/job-wait.sh ${JOB_ID}
```

Comment:
I'm a complete novice when it comes to CodeBuild, but wouldn't the script path be relative to the parent of the `buildspec` directory, meaning just `scripts/job-wait....`?

Reply:
It always starts at the root of the project directory, in this case `contrib/`, so all paths are relative to that.